python/vowpalwabbit/sklearn_vw.py

```diff
@@ -673,6 +673,74 @@ class VWRegressor(VW, RegressorMixin):
     pass
 
 
+class VWMultiClassifier(ClassifierMixin, VW):
+    """Vowpal Wabbit MultiClassifier model """
+
+    def __init__(self, **params):
+
+        params['probabilities'] = True
+        super(VWMultiClassifier, self).__init__(**params)
+
+    def predict(self, X):
+        """Predict class labels for samples in X.
+
+        Parameters
+        ----------
+        X : {array-like, sparse matrix}, shape = [n_samples, n_features]
+            Samples.
+
+        Returns
+        -------
+        C : array, shape = [n_samples]
+            Predicted class label per sample.
+
+        Examples
+        --------
+        >>> import numpy as np
+        >>> X = np.array([ [10, 10], [8, 10], [-5, 5.5], [-5.4, 5.5], [-20, -20],  [-15, -20] ])
+        >>> y = np.array([1, 1, 2, 2, 3, 3])
+        >>> from vowpalwabbit.sklearn_vw import VWMultiClassifier
+        >>> model = VWMultiClassifier(oaa=3, loss_function='logistic')
+        >>> model.fit(X, y)
+        >>> model.predict(X)
+        """
+
+        scores = self.predict_proba(X)
+        indices = scores.argmax(axis=1)+1
```

Comment:
Can you explain why this is needed?

Reply:
I see, this selects the largest score in axis 1 for each list if probabilities then adds 1 to make it start from 1.